static/js/browser2/out_arc_section.tsx

```diff
@@ -191,43 +210,53 @@ export class OutArcSection extends React.Component<
   };
 
   private fetchDataFromTriples(): void {
-    axios.get("/api/browser/triples/" + this.props.dcid).then((resp) => {
-      const triplesData = resp.data;
-      const outArcs = triplesData.filter(
-        (t) => t.subjectId === this.props.dcid
-      );
-      const outArcsByPredProv: OutArcData = {};
-      for (const outArc of outArcs) {
-        const predicate = outArc.predicate;
-        if (IGNORED_OUT_ARC_PROPERTIES.has(predicate)) {
-          return;
-        }
-        if (!outArcsByPredProv[predicate]) {
-          outArcsByPredProv[predicate] = {};
-        }
-        const outArcsOfPredicate = outArcsByPredProv[predicate];
-        const provId = outArc.provenanceId;
-        if (!(provId in outArcsOfPredicate)) {
-          outArcsOfPredicate[provId] = [];
-        }
-        let valueText = "";
-        let valueDcid: string;
-        if (outArc.objectId) {
-          valueText = outArc.objectName ? outArc.objectName : outArc.objectId;
-          valueDcid = outArc.objectId;
-        } else {
-          valueText = outArc.objectValue;
+    loadSpinner(LOADING_CONTAINER_ID);
+    axios
+      .get("/api/browser/triples/" + this.props.dcid)
+      .then((resp) => {
+        const triplesData = resp.data;
+        const outArcs = triplesData.filter(
+          (t) => t.subjectId === this.props.dcid
+        );
+        const outArcsByPredProv: OutArcData = {};
+        for (const outArc of outArcs) {
+          const predicate = outArc.predicate;
+          if (IGNORED_OUT_ARC_PROPERTIES.has(predicate)) {
+            return;
```

Comment:
need to remove spinner() before return?

Reply:
this return actually should be a continue, which I've updated